src/cli_validator/cmd_meta/validator.py

```diff
@@ -26,6 +27,22 @@ def validate_command(self, command, comments=True):
         parser = self.build_parser(meta)
         namespace = parser.parse_args(cmd.parameters)
 
+        if namespace.ids is not None:
+            for param in meta['parameters']:
+                if 'id_part' in param:
+                    continue
+                else:
+                    if 'required' in param and namespace.__getattribute__(param['name']) is None:
+                        raise ValidateFailureException('Some arguments are required)
```

Comment:
Please make sure that you close the quotation marks correctly.

```suggestion
                        raise ValidateFailureException('the following arguments are required: {}'.format('/'.join(param['options'])))
```

We could display the missing parameter in error message.

Reply:
Sure. I'll create a list where I store all the missing parameters and raise the error.